instrumentation/go.mongodb.org/mongo-driver/config.go

```diff
@@ -25,30 +25,31 @@ const (
 
 // Config is used to configure the mongo tracer.
 type Config struct {
+	TraceProvider trace.Provider
+
 	Tracer trace.Tracer
```

Comment:
```suggestion
	TracerProvider trace.Provider
```

Reply:
Look like Mongo's `monitor` still needs it.

https://github.com/open-telemetry/opentelemetry-go-contrib/blob/6b661766df2c26b415e003ac00f5aeae094c3403/instrumentation/go.mongodb.org/mongo-driver/mongo.go#L57